apps/src/javalab/Neighborhood.js

```diff
@@ -60,6 +60,7 @@ export default class Neighborhood {
       this.onOutputMessage(
         `${STATUS_MESSAGE_PREFIX} ${javalabMsg.startingPainter()}`
       );
+      this.onNewlineMessage();
```

Comment:
I can't put a comment on a non-edited line, but since we added this newline here, we should remove the one on line 77 so we don't have two blank lines between neighborhood executions